Tools/jsdoc/cesium_template/tmpl/indexLayout.tmpl

```diff
@@ -17,7 +17,7 @@
 
 <div class="nav">
     <div class="menu">
-        <a href="http://cesiumjs.org/" target="_top"><img id="cesium-logo" src="images/CesiumLogo.png" class="cesiumLogo"></a>
+        <a href="http://cesiumjs.org/" target="_top"><img id="cesium-logo" src="Images/CesiumLogo.png" class="cesiumLogo"></a>
```

Comment:
This file moved.  `static/images` doesn't exist anymore on this branch.

The problem was, `cesium_template/static/**` were all lowercase, including `images`.  But, out in the main source tree, `Documentation` contained all uppercase, `Contributors`, `Images`, and `Schemas`.  The `Images` folder was copied over top of the `images` folder in the build output, which created separate folders on case-sensitive systems, but merged contents on insensitive systems.

What I've done is moved the lone file from `static/images` to `Documentation/Images`, so that all image files get the uppercase `I`, and the lowercase folder is completely gone.

Reply:
Perfect!